elyra/pipeline/processor_local.py

```diff
@@ -76,7 +76,7 @@ def process(self, pipeline):
                                        operation_name=operation.name,
                                        duration=(time.time() - t0))
             except Exception as ex:
-                raise RuntimeError(f'Error processing operation {operation.name}.') from ex
+                raise RuntimeError(f'Error processing operation {operation.name}: {str(ex)}.') from ex
```

Comment:
This is side-effecting all errors during local processing.

I think we need to define exactly how much "detail" should be conveyed into the message title w/o parsing the traceback.